chia/util/initial-config.yaml

```diff
@@ -636,6 +641,14 @@ solver:
   # Enable or disable UPnP port forwarding
   enable_upnp: False
 
+  # Node IDs of trusted solver peers, only these can connect by default
+  trusted_peers:
```

Comment:
is this how we trust other kinds of peers too? It doesn't seem safe to just base it on IP and port. On a multi-user system, for instance, any other user would automatically be trusted by this mechanism.